src/applications/disability-benefits/all-claims/content/form0781.jsx

```diff
@@ -144,22 +196,30 @@ export const mentalHealthSupportAlert = () => {
   return (
     <va-alert-expandable
       status="info"
-      trigger="Learn how to get mental health help now"
+      trigger="Get mental health and military sexual trauma support anytime"
     >
       <p>
-        We understand that some of the questions may be difficult to answer. If
-        you need to take a break and come back to your application, your
-        information will be saved.
+        We can connect you with free, confidential support for mental health
+        care or military sexual trauma anytime. We can connect you with support
+        even if you don’t file a claim or disability compensation or you aren’t
```

Comment:
Typo in original Figma comps. See [message](https://dsva.slack.com/archives/C0983077SVA/p1753995223204369) discussing correction.

```suggestion
        even if you don’t file a claim for disability compensation or you aren’t
```